### PR TITLE
Fixes example snippet in Array.swift adding missing parameter label

### PR DESCRIPTION
**Pull request by valeriyvan**

### CODE REVIEW

**Comment by xwu** [comment]
How about we demonstrate good Swift API naming practice here and consider calling it `cacheImages(withNames:)` or something like that.

**Comment by valeriyvan** [comment]
Fixed

File: stdlib/public/core/Array.swift [hunk 1]
```diff
@@ -852,14 +852,14 @@ extension Array: RangeReplaceableCollection {
   /// `LazyMapCollection<Dictionary<String, Int>, Int>` to a simple
   /// `[String]`.
   ///
-  ///     func cacheImagesWithNames(names: [String]) {
+  ///     func cacheImagesWithNames(withNames: [String]) {
```
Comment (xwu) [inline stdlib/public/core/Array.swift:855]:
```suggestion
  ///     func cacheImages(withNames names: [String]) {
```

File: stdlib/public/core/Array.swift [hunk 2]
```diff
@@ -852,14 +852,14 @@ extension Array: RangeReplaceableCollection {
...
   ///         // custom image loading and caching
   ///      }
   ///
   ///     let namedHues: [String: Int] = ["Vermillion": 18, "Magenta": 302,
   ///             "Gold": 50, "Cerise": 320]
   ///     let colorNames = Array(namedHues.keys)
-  ///     cacheImagesWithNames(colorNames)
+  ///     cacheImagesWithNames(withNames: colorNames)
```
Comment (xwu) [inline stdlib/public/core/Array.swift:862]:
```suggestion
  ///     cacheImages(withNames: colorNames)
```

**Comment by xwu** [comment]
cc @natecook1000

**Comment by xwu** [comment]
@swift-ci smoke test

**Review by natecook1000 — APPROVED** [review]

**Comment by natecook1000** [comment]
@swift-ci Please smoke test macOS platform

**Comment by xwu** [comment]
@swift-ci Please smoke test macOS platform

**Pull request merged**